src/main/kotlin/blackjack/view/InputView.kt

```diff
@@ -26,4 +26,14 @@ object InputView {
         }
         return input
     }
+
+    fun getBettingAmount(playerName: String): Int {
+        println("Enter $playerName's betting amount:")
+        val inputString = readLine()?.trim() ?: ""
```

Comment:
```suggestion
        val inputString = readlnOrNull()?.trim() ?: ""
```

Consider using this to avoid a potential `NullPointerException`.